src/main/java/org/fcrepo/client/FcrepoClient.java

```diff
@@ -152,6 +154,17 @@ public HistoricMementoBuilder createMemento(final URI url, final String mementoD
         return new HistoricMementoBuilder(url, this, mementoDatetime);
     }
 
+    /**
+     * Make a POST request to create a new Transaction
+     *
+     * @param url the base url of the Fedora repository
+     * @return ???
+     */
+    public PostBuilder startTransaction(final URI url) {
```

Comment:
I originally had additional methods in FcrepoClient which returned builders for each similar to how `startTransaction` is now. I wasn't really sure how I felt about them which is why they were omitted when I put up the PR. Good to know about the Transaction API being more limited than the general PUT/POST operations, that's not something I had considered.  

I do like the idea of 3 - I was considering something similar initially but didn't end up going too far with it. I'll try this out again to see how it feels; since you've mentioned it I'm not as hesitant to add it in. At the very least I'll get a new `TransactionBuilder` in which limits what can be done.